source/NPC.cpp

```diff
@@ -241,6 +273,45 @@ void NPC::Save(DataWriter &out) const
 
 
 
+// Update to spawn conditions for if this NPC can be placed.
+void NPC::CanSpawn(const PlayerInfo &player)
+{
+	// Only check the toSpawn conditions the first time this function is called.
```

Comment:
Does this mean that an NPC only has 1 single chance to spawn?
I would expect the CanSpawn to be checked multiple times during the mission (as long as the NPC didn't spawn yet).